eudemo/eudemo/equilibria/_designer.py

```diff
@@ -133,8 +133,8 @@ def __init__(
     ):
         super().__init__(params, build_config)
         self.file_path = self.build_config.get("file_path", None)
-        self.diagnostic_plotting = self.build_config.get(
-            "diagnostic_plotting", PicardDiagnosticOptions()
+        self.diagnostic_plotting = PicardDiagnosticOptions(
+            *self.build_config.get("diagnostic_plotting", {})
```

Comment:
dictionary -- 2 stars not one
```suggestion
            **self.build_config.get("diagnostic_plotting", {})
```